docs/userguide/recovery_device.rst

```diff
@@ -0,0 +1,80 @@
+.. Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+.. _doc_userguide_recovery_device:
+
+===============
+Recover devices
+===============
+
+A *recovery device* is a backup of an existing device. If you lose access to Parsec (you forget the password), the recovery device will allow you to recreate a new device and set a new password.
```

Comment:
Forgetting the password is not the only use case, it could also be that the parsec keys have been lost (e.g the hard drive broke or has been formatted)

Reply:
```suggestion
A *recovery device* is a backup of an existing device. 
If you lose access to Parsec (you forget the password, the Parsec keys has been lost, etc), 
the recovery device will allow you to recreate a new device and set a new password.
```